docs/blueprints/EP036.rst

```diff
@@ -130,16 +173,35 @@ This event should sent before any NApp sends its first flow to be published.
     "coloring": {"base": 0}
   }
 
-Deploying pipeline
+
+
+Endpoint API

```

Comment:
Great API endpoints design/spec. You're using `v2` versioned routes. It's great you versioned the routes, but we probably should be starting with `v1`, right?